src/Components/Home-page-Card/Data.js

```diff
@@ -0,0 +1,29 @@
+const Data_one=[
+    {
+        image:"/image/shoe3.jpeg",
+        subtitle:"upto 10% off"
+    },
+    {
+        image:"/image/shoe4.jpeg",
+        subtitle:"upto 20% off"
+    },
+    {
+        image:"/image/shoe5.jpeg",
+        subtitle:"upto 15% off"
+    },
+]
+const Data_Two=[
+    {
+        image:"/image/shoe13.webp",
```

Comment:
suggested that use proper name like if you have top brand search then use a name like topBrandData